### PR TITLE
 Assignment Operators

### PR DESCRIPTION
**Pull request by deepankkartikey**
Pull request for https://github.com/avsingh999/JavaScript-for-Developer/issues/8

### CODE REVIEW

File: assignmentOperators.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+Examples
+
+The `=` assignment operator assigns a value to a variable.
+```
+var x = 10;
+console.log(x) //10
+```
+The `+=` assignment operator adds a value to a variable.
```
Comment (jsgiant) [inline assignmentOperators.md]: I think It is better to describe it like:
The `+=` assignment operator adds the value of the variable and value given in the RHS and it assigns the result to the variable. Is it looking good?

Comment (deepankkartikey) [reply assignmentOperators.md]: Addressed this

**Review by jsgiant — COMMENTED** [review]
Good work @deepank120896 ! @avsingh999  Checkout this, If you got enough you can merge it.

**Comment by avsingh999** [comment]
Hi @deepank120896  @jsgiant  Thanks ❤️

**Pull request merged**